force-app/main/default/classes/CRLP_ResetRollupFieldsQueueable_TEST.cls

```diff
@@ -0,0 +1,160 @@
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @group Rollups
+* @group-content ../../ApexDocContent/Rollups2.htm
+* @description Tests for queueable job to clear rollup values for records with no associated details
+* which have been previously calculated in skew mode.
+*/
+@isTest(IsParallel=true)
+private class CRLP_ResetRollupFieldsQueueable_TEST {
+
+    @IsTest
+    private static void shouldClearGAUTotalsWhenRecordIdIsUnhandled() {
+        Integer amount = 90;
+        Integer count = 1;
+
+        General_Accounting_Unit__c gau =
+            new General_Accounting_Unit__c(Total_Allocations__c = amount, Total_Number_of_Allocations__c = count);
+        insert gau;
+
+        mockRollupCMTValues();
+
+        CRLP_GAU_BATCH gauBatch = new CRLP_GAU_BATCH(new List<Id>{gau.Id}, null);
+
+        Test.startTest();
+        gauBatch.clearUnhandledRollupTotals();
+        Test.stopTest();
+
+        List<General_Accounting_Unit__c> generalAccountingUnits =
+            [SELECT Id, Total_Allocations__c, Total_Number_of_Allocations__c FROM General_Accounting_Unit__c];
+
+        System.assertEquals(0, generalAccountingUnits[0].Total_Allocations__c,
+            'Total Allocations should be reset to 0');
+        System.assertEquals(0, generalAccountingUnits[0].Total_Number_of_Allocations__c,
+            'Total Number of Allocations should be reset to 0');
+    }
+
+    @IsTest
+    private static void shouldNotClearGAUTotalsWhenRecordIdIsHandled() {
+        Integer amount = 90;
+        Integer count = 1;
+
+        General_Accounting_Unit__c gau =
+            new General_Accounting_Unit__c(Total_Allocations__c = amount, Total_Number_of_Allocations__c = count);
+        insert gau;
+
+        mockRollupCMTValues();
+
+        CRLP_GAU_BATCH gauBatch = new CRLP_GAU_BATCH(new List<Id>{gau.Id}, null);
+        gauBatch.handledParentRecordIds.addAll(gauBatch.recordIds);
+
+        Test.startTest();
+        gauBatch.clearUnhandledRollupTotals();
+        Test.stopTest();
+
+        List<General_Accounting_Unit__c> generalAccountingUnits =
+        [SELECT Id, Total_Allocations__c, Total_Number_of_Allocations__c FROM General_Accounting_Unit__c];
+
+        System.assertEquals(amount, generalAccountingUnits[0].Total_Allocations__c,
+            'Total Allocations should retain previous value');
+        System.assertEquals(count, generalAccountingUnits[0].Total_Number_of_Allocations__c,
+            'Total Number of Allocations should retain previous value');
+    }
+
+    @IsTest
+    private static void shouldClearSkewModeFlagForContact() {
+        List<Contact> contacts = new List<Contact>{new Contact(CustomizableRollups_UseSkewMode__c = true)};
```

Comment:
The `mockRollupCMTValues()` method will need to be called in all test methods well because the `clearUnhandledRollupTotals()`  method will use the rollup services to build a query that retrieves all target fields from the rollup definitions - and then clear those fields. If there are no rollup definitions, then the call to the `clearUnhandledRollupTotals()` method won't actually do anything.

Reply:
ahh...  totally makes sense.  thanks!!